### PR TITLE
[Notts Police] Council referral functionality

### PR DESCRIPTION
**Pull request by davea**
[skip changelog]

### CODE REVIEW

File: templates/email/nottinghamshirepolice/submit.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+[%
+
+email_summary = "A new problem in your area has been reported by a " _ site_name _ " user.";
+email_footer = "If there is a more appropriate email address for messages about " _ ( report.get_extra_metadata('original_category') OR report.category ) _ ", please let us know. This will help improve the service for local people. We also welcome any other feedback you may have.";
```
Comment (dracos) [inline templates/email/nottinghamshirepolice/submit.html:4]: Do we want to include this message, we probably don't want this to happen for this - and they can work out how to contact the police if they have an issue, I assume.

File: templates/email/nottinghamshirepolice/submit.txt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+View OpenStreetMap of this location: [% osm_url %]
+
+[% closest_address %]----------
+
+Replies to this email will go to the user who submitted the problem.
+
+[% signature %]
+
+If there is a more appropriate email address for messages about
```
Comment (dracos) [inline templates/email/nottinghamshirepolice/submit.txt:45]: Ditto

**Review by dracos — CHANGES_REQUESTED** [review]
Looks good, if we're going to have custom template, might as well customise it a bit more.

**Comment by davea** [comment]
Thanks, good thinking. I've added a fixup for that, and a new commit that improves the inspector form layout a bit for this use case:

image

**Review by dracos — APPROVED** [review]
Looks good

**Pull request merged**